File: PRAK304/PRAK304-2210817310013-RyanMuhammadIrfan.c
#include <stdio.h>

int main()
{
    int bilangan;

    printf("Masukan bilangan:");
    scanf("%d", &bilangan);

    if (bilangan == 0)
    {
        printf("Nol");
    }

    else if (bilangan >= 1 && bilangan < 10)
    {
        printf("Satuan");
    }

    else if (bilangan >= 10 && bilangan < 20)
    {
        printf("Belasan");
    }

    else if (bilangan >= 20 && bilangan < 100)
    {
        printf("Puluhan");
    }

    else
    {
        printf("Anda Menginput Melebihi Limit Bilangan");
    }
}

File: PRAK304/PRAK304-2210817310013-Ryan Muhammad Irfan.py
bilangan = int(input("Masukan bilangan:"))


if bilangan == 0:
    print("Nol")

elif bilangan >= 1 and bilangan < 10:
    print("Satuan")

elif bilangan >= 10 and bilangan < 20:
    print("Belasan")

elif bilangan >= 20 and bilangan < 100:
    print("Puluhan")

else:
    print("Anda Menginput Melebihi Limit Bilangan")
